wikimedia/wikipedia-ios
# Fix app_session event server error

**Phabricator:** 
Found when investigating https://phabricator.wikimedia.org/T335549#8924489

### Notes
I noticed we were getting this error message from the server when we had no `page_load_latency_ms` data to send:

Request:
```
{
  "app_install_id": "01C6A34E-C0E5-4C80-AD7A-9710538E2D61",
  "length_ms": 200893,
  "dt": "2023-06-13T19:54:48Z",
  "primary_language": "en",
  "meta": {
    "stream": "app_session",
    "id": "749C70F4-99D1-4682-8D56-3E11A8BBB7B7"
  },
  "$schema": "/analytics/mobile_apps/app_session/1.0.0",
  "app_session_id": "5f01523b74a95195bcbc",
  "is_anon": false
}
```

Response:
```
{
  "invalid": [
    {
      "status": "invalid",
      "event": {
        ...
      },
      "context": {
        "errors": [
          {
            "keyword": "required",
            "dataPath": "",
            "schemaPath": "#/required",
            "params": {
              "missingProperty": "session_data"
            },
            "message": "should have required property 'session_data'"
          }
        ],
        "errorsText": "'' should have required property 'session_data'"
      }
    }
  ]
}
```

### Test Steps
1. Launch app, inspect calls to server with a proxy.
2. Turn on analytics via Settings or Onboarding
3. Without visiting any articles, turn off the analytics toggle and wait for the `app_session` event to POST
4. Confirm event posts successfully with an empty `session_data` object now, instead of a missing one.
